pdr_backend/lake/lake_validate.py

```diff
@@ -17,171 +12,6 @@
 
 @enforce_types
 class LakeValidate(LakeInfo):
```

Comment:
should we delete this?

Reply:
I don't think so. We still have the same CLI validation independently of the dashboard. CLI describe command still works exactly as before (+ your timestr changes). It's only the dashboard that combines Info + Validation as a full report on the ETL.